internal/org/model/member.go

```diff
@@ -0,0 +1,21 @@
+package model
+
+import es_models "github.com/caos/zitadel/internal/eventstore/models"
+
+type OrgMember struct {
+	es_models.ObjectRoot
+	UserID string
+	Roles  []string
+}
+
+func NewOrgMember(orgID, userID string) *OrgMember {
+	return &OrgMember{ObjectRoot: es_models.ObjectRoot{AggregateID: orgID}, UserID: userID}
+}
+
+func NewOrgMemberWithRoles(orgID, userID string, roles ...string) *OrgMember {
+	return &OrgMember{ObjectRoot: es_models.ObjectRoot{AggregateID: orgID}, UserID: userID, Roles: roles}
+}
+
+func (member *OrgMember) IsValid() bool {
+	return member.AggregateID != "" && member.UserID != "" && len(member.Roles) != 0
```

Comment:
do we always need roles? what if i remove last role?

Reply:
I asked me the same question. Is a member still a member if it has no roles?

Reply:
did it analog project (no check if roles is 0)